src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -206,6 +206,8 @@ export const UserDetailsForm: React.FC<Props> = ({
         name="ssn"
         id="ssn"
         type="text"
+        helperText={'info message'}
```

Comment:
Is this just to demo the helperText?

Reply:
yes. just for demo . will revert back after we validate the PR.